src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,48 @@
-export const Tabs = () => {};
+type Tab = {
+  id: string,
+  title: string,
+  content: string,
+};
+
+type TabProps = {
+  tabs:Tab[],
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected } : TabProps) => {
+  const selectedTab = tabs
+    .find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={selectedTab.id === tab.id ? 'is-active' : ''}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => {
```

Comment:
You might consider using the preventDefault() method within the onClick attribute to prevent URL change when the user clicks a tab